### DISCUSSION TITLE
Export to PDF

### DISCUSSION THREAD

**Discussion by Splashman777** [singerdmx/flutter-quill#362] [General]
I have no issue with this package but am wondering if anyone has successfully exported formatted text to a PDF. I've used the Flutter PDF package but it seems to have a limited set of widgets that it can display. Therefore, I have not yet found a way to maintain text formatting when displaying as a PDF. Converting to plain text works but I'd like to keep the formatting if possible.

Any idea how to accomplish this? Thanks!

**Comment by MohammedSalehelShazly** [comment]
any updates?

**Comment by andynvt** [comment]
any updates?

**Comment by CatHood0** [comment]
Just use [flutter_quill_to_pdf](https://github.com/CatHood0/flutter_quill_to_pdf)
